src/data_validation.ts

```diff
@@ -1,20 +1,23 @@
 import { MethodRegistrationBase, ArgRequiredOptions } from "./decorators";
-import { DBOSContextImpl } from "./context";
+import { DBOSContextImpl, getCurrentDBOSContext } from "./context";
 import { DBOSDataValidationError } from "./error";
 
 export function validateMethodArgs<Args extends unknown[]>(methReg: MethodRegistrationBase, args: Args)
 {
     let opCtx : DBOSContextImpl | undefined = undefined;
+    if (!methReg.passContext) {
+        opCtx = getCurrentDBOSContext() as DBOSContextImpl;
+    }
 
     const validationError = (msg:string) => {
         const err = new DBOSDataValidationError(msg);
-        opCtx?.span.addEvent("DataValidationError", { message: err.message });
+        opCtx?.span?.addEvent("DataValidationError", { message: err.message });
```

Comment:
When can this span be missing?

Reply:
I see. But in practice, given how we use `validationError`, this should never happen (we always use `validationError` for arguments number 2 and more, and `opCtx` is set either when entering `validateMethodArgs`, either when checking the first argument.)